scratchapi2/meta.py

```diff
@@ -1,118 +0,0 @@
-#pylint: disable=trailing-whitespace
-"""
-Scratch API Meta
-
-It contains some functions that are not categorized.
-Contains:
-* API Information
-* Search
-* Statistics
-* Scratch Information
-
-NOTE: This feature is implemented in Misc:
-* Translate
-
-Anything put on Front Page is implemented in FrontPage.
-"""
-#pylint: enable=trailing-whitespace

```

Comment:
Also, why did you disable trailing whitespace? Trailing whitespace is a terrible idea!

Reply:
Pylint warned me even the program contains ONLY THAT PYDOC!